packages/jsii-rosetta/lib/languages/python.ts

```diff
@@ -735,7 +735,8 @@ export class PythonVisitor extends DefaultVisitor<PythonLanguageContext> {
         case 'list':
           return `list of ${doRender(jsiiType.elementType)}`;
         case 'namedType':
-          return jsiiType.name;
+          // in this case, the fallback will hold more information than jsiiType.name
+          return fallback;
```

Comment:
I don't love this, but the information we want is in `fallback`, which comes from `node.type.getText()`. I fiddled around with things to see if I could somehow isolate the module name from some other variable, but there isn't anything besides `getText()` to my knowledge.

Reply:
Why don't we read, interpret and format `jsiiType.fqn` ourselves?